projects/dotcms-field-elements/src/components/dot-radio/dot-radio.tsx

```diff
@@ -0,0 +1,130 @@
+import { Component, Element, Event, EventEmitter, Method, Prop, State } from '@stencil/core';
+import Fragment from 'stencil-fragment';
+import { DotFieldStatus, DotFieldStatusEvent, DotFieldValueEvent, DotOption } from '../../models';
+import {
+    getClassNames,
+    getDotOptionsFromFieldValue,
+    getErrorClass,
+    getOriginalStatus,
+    getTagError,
+    getTagHint,
+    getTagLabel,
+    updateStatus
+} from '../../utils';
+
+@Component({
+    tag: 'dot-radio',
+    styleUrl: 'dot-radio.scss'
+})
+export class DotRadioComponent {
+    @Element() el: HTMLElement;
+
+    @Prop({ mutable: true })
+    value: string;
+    @Prop() name: string;
+    @Prop() label: string;
+    @Prop() hint: string;
+    @Prop() required: boolean;
+    @Prop() disabled = false;
+    @Prop() requiredmessage: string;
+    @Prop() options: string;
+
+    @State() _options: DotOption[];
```

Comment:
Why we have `options` and `_options`?

Reply:
`options` is the user input , `_options` is an array of `DotOption` to handle the values internally, just like `checkbox`